projects/plugins/crm/includes/ZeroBSCRM.Core.php

```diff
@@ -1353,20 +1356,9 @@ public function includes() {
 	 */
 	private function init_hooks() {
 
-		// General activation hook: DB check, role creation
-		register_activation_hook( ZBS_ROOTFILE, array( $this, 'install' ) );
-
-		add_action( 'activated_plugin', array( $this, 'activated_plugin' ) );
-
 		// Pre-init Hook
 		do_action( 'before_zerobscrm_init' );
 
-		// After all the plugins have loaded (THESE FIRE BEFORE INIT)
-		add_action( 'plugins_loaded', array( $this, 'load_textdomain' ) ); // } Translations
```

Comment:
Should we chop `ZeroBSCRM::load_textdomain` now?

Reply:
we should!